components/PersonView/PersonView.jsx

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect } from 'react';
+import { getResident } from 'utils/api/residents';
+
+const PersonView = ({ personId }) => {
+  const [person, setPerson] = useState();
+
+  const getPerson = async (personId) => {
+    try {
+      const data = await getResident(personId);
+      setPerson(data);
+    } catch {
+      console.log('error');
```

Comment:
We need to handle this error, for example showing a custom `ErrorMessage`

Reply:
I can see that other components use <ErrorMessage label ={error} />  But I'm not clear where we define the error.

Reply:
Something like https://github.com/LBHackney-IT/lbh-social-care/blob/develop/components/Cases/Cases.jsx#L20.

The catch should intercept the error returned from the server `catch(e)` and handle it.